core-java/src/main/java/com/baeldung/algorithms/annealing/City.java

```diff
@@ -2,21 +2,35 @@
 
 import lombok.Data;
 
-@Data
-public class City {
+@Data public class City {
```

Comment:
Lack of getters was not a problem here. The @Data annotation that is present here is taking care of this if you enable annotation processing in your IDE